templates/algob-react-boilerplate/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Please change the title and add more information that this is a template for Algorand Builder web Dapp

Reply:
OK.